app/views/audits/show.html.erb

```diff
@@ -31,22 +31,22 @@
       </div>
       <div class="row">
         <strong class="col-md-2"><%= _("Audit Time:")%></strong>
-        <div class="col-md-10"><%= date_time_absolute(@audit.created_at) %></div>
+        <div class="col-md-10"><%= audit_time(@audit) %></div>
       </div>
       <div class="row">
         <strong class="col-md-2"><%= _("User Name:")%></strong>
-        <div class="col-md-10"><%= @audit.username %></div>
+        <div class="col-md-10"><%= audit_user(@audit) %></div>
```

Comment:
Note that this leads to all audits by this user, not the user itself, not sure if this was intentional or not